README.md

```diff
@@ -157,6 +157,78 @@ environment. You can plott it in a more human-readable way with the
 [puppet/format](https://github.com/voxpupuli/puppet-format?tab=readme-ov-file#puppet-format)
 modules.
 
+
+The plan `pe_status_check::agent_state_summary` provides you a hash with all nodes, grouped by failure state:
+
+```json
+{
+  "noop": [ ],
+  "corrective_changes": [ ],
+  "used_cached_catalog": [ ],
+  "failed": [ ],
+  "changed": [ "student2.local" ],
+  "unresponsive": [ "student3.local", "student4.local", "student1.local", "login.local" ],
+  "responsive": [ "pe.bastelfreak.local"],
+  "unhealthy": [ "student2.local", "student3.local", "student4.local", "student1.local", "login.local" ],
+  "unhealthy_counter": 5,
+  "healthy": [ "pe.bastelfreak.local" ],
```

Comment:
The potential result size of _healthy_ would be my concern. Would it be reasonable to exclude the array by default (or at all)?

Another option would be the summary is counters by default, a detailed result - which includes everything but healthy array?

If you really want the healthy nodes, maybe a verbose parameter?

Reply:
ah good idea. I only tested it in a small environment, but for larger deployments the outpu would kill your terminal. I will adjust it.

Reply:
@h0tw1r3 can you take another look please?